package_installs.R

```diff
@@ -19,10 +19,14 @@ pkgs <- as.character(packages$Package)
 M <- min(M, length(pkgs))
 
 do_one <- function(repo, pkg){
+  debug <- FALSE
+  if (pkg == "Cairo" || pkg == "repr" || pkg == "IRkernel") {
```

Comment:
Could you write a list at the beginning of the file with the packages that must be installed? And add comments about them? Then check here that it's in the list?

Reply:
This is a temporary addition until we figure out why the build is broken.  It seems not worth the time to define a list for something that will hopefully be deleted in a few hours, especially since I don't know R and that would increase the chance that I'd get something wrong.